README.md

```diff
@@ -160,6 +160,22 @@ const menuTemplate = new MenuTemplate<MyContext>((ctx) => {
 });
 ```
 
+### Can I use `entities` like with `grammyjs/parse-mode` in the message body?
+
+Yes, you can pass the option `entities`.
```

Comment:
Thought about linking to the Repo like the example above also links to `telegram-format`. Also, the current sentence is kinda self explaining so it can be removed in my opinion. What do you think?

```suggestion
### Can I use `entities` in the message body?

Also see: [`grammyjs/parse-mode`](https://github.com/grammyjs/parse-mode)
```